packages/hooks/src/test/index.test.js

```diff
@@ -0,0 +1,4 @@
+/**
+ * Internal Dependencies
+ */
+import { doAction, applyFilters, addAction, addFilter, doingAction, doingFilter, didAction, didFilter, hasAction, hasFilter, removeAllActions, removeAllFilters, currentFilter } from '../';
```

Comment:
I'll work on cleaning this up and maybe splitting out the generator code into files/export/import.